pkg/cloud/cloud.go

```diff
@@ -51,6 +51,10 @@ const (
 	VolumeTypeSC1 = "sc1"
 	// VolumeTypeST1 represents a throughput-optimized HDD type of volume.
 	VolumeTypeST1 = "st1"
+	// VolumeTypeSBG1 represents a capacity-optimized HDD type of volume.
```

Comment:
np: Can you add something like `(only for SBE devices)` to the comments, to make it easier for readers to know why we're adding these types.

Reply:
Ack, made changes.